src/test/kotlin/com/noumenadigital/npl/cli/BinaryCommandsIT.kt

```diff
@@ -5,19 +5,36 @@ import com.noumenadigital.npl.cli.TestUtils.runCommand
 import io.kotest.core.spec.style.FunSpec
 import io.kotest.matchers.equals.shouldBeEqual
 import io.kotest.matchers.shouldBe
+import java.io.File
+import javax.xml.parsers.DocumentBuilderFactory
 
 class BinaryCommandsIT :
     FunSpec({
-        test("version command should return correct value") {
+        test("version command should return correct version based on test mode") {
+            val pomFile = File("pom.xml")
+            val dbFactory = DocumentBuilderFactory.newInstance()
+            val dBuilder = dbFactory.newDocumentBuilder()
+            val doc = dBuilder.parse(pomFile)
+            doc.documentElement.normalize()
+
+            val pomVersion =
+                doc.documentElement
+                    .getElementsByTagName("version")
+                    .item(0)
+                    .textContent
```

Comment:
This is pretty damn ugly, but I couldn't think of a nicer way.

One alternative would be to update the test every time, but I hate that.

This way we do compare against the "real" version, and the test is as real as it gets.